nmostesting/suites/IS1301Test.py

```diff
@@ -0,0 +1,316 @@
+
+    def set_up_tests(self):
+        """
+        FAKE_ORIG = {
+            'description': 'fake_orig_desc',
+            'label': 'fake_orig_label',
+            'tags': {
+                'location': ['fake_location']
+            }
+        }
+        for resource in RESOURCES:
+            url = "{}{}{}".format(self.annotation_url, 'node/', resource)
+            TestHelper.do_request("PATCH", url, json=FAKE_ORIG)
+        """
+        pass
+
+    def get_resource(self, url):
+        """ Get a resource """
+
+        valid, r = self.do_request("GET", url)
+        if valid and r.status_code == 200:
+            try:
+                return True, r.json()
+            except Exception as e:
+                return False, e.msg
+        else:
+            return False, "GET  Resquest FAIL"
+
+    def compare_resource(self, object, new, resp):
+        """ Compare string values (or "tags" dict) """
+
+        if new[object] is None:  # this is a reset, new is null, skip
+            return True, ""
+
+        if object == "tags":  # tags needs to be stripped
+            resp[object] = strip_tags(resp[object])
+            if not TestHelper.compare_json(resp[object], new[object]):
+                return False, f"{object} value FAIL"
+
+        elif resp[object] != new[object]:
+            return False, f"{object} value FAIL"
+
+        return True, ""
+
+    def set_resource(self, url, node_url, new, prev):
+        """ Patch a resource with one ore several object values """
+
+        object = list(new.keys())[0]
+        valid, resp = TestHelper.do_request("PATCH", url, json=new)
+        if not valid:
+            return False, "PATCH Resquest FAIL"
+
+        # pause to accomodate update propagation
+        time.sleep(0.1)
+
+        # re-GET
+        valid, resp = self.get_resource(url)
+        if not valid:
+            return False, "Get Resquest FAIL"
+        # check PATCH == GET
+        valid, msg = self.compare_resource(object, new, resp)
+        if not valid:
+            return False, f"new {msg}"
+        # check that the version (timestamp) has increased
+        if get_ts_from_version(prev['version']) >= get_ts_from_version(resp['version']):
+            return False, "new version (timestamp) FAIL"
+
+        # validate that it is reflected in IS04
+        valid, node_resp = self.get_resource(node_url)
+        if not valid:
+            return False, "GET IS-04 Node FAIL"
+        # check PATCH == GET
+        valid, msg = self.compare_resource(object, new, resp)
+        if not valid:
+            return False, f"new IS-04/node/.../ {msg}"
+        # check that the version (timestamp) has increased
+        if get_ts_from_version(node_resp['version']) != get_ts_from_version(resp['version']):
+            return False, "new IS-04/node/.../version (timestamp) FAIL"
+
+        return True, resp
+
+    def log(self, msg):
+        print(msg)

```

Comment:
Generally the test suites don't print to the console as the console reporting is handled by the testing framework, so the "logging" is best removed.